app/helpers/routing_helper.rb

```diff
@@ -0,0 +1,57 @@
+require 'net/http'
+require 'json'
+
+module RoutingHelper
+  def self.seconds_to_minsec(sec)
+    format("%<minutes>.2d:%<seconds>.2d", minutes: sec / 60 % 60, seconds: sec % 60)
```

Comment:
This can only handle durations up to 60 minutes, right? Maybe a bit weird, but should be fine because HPI is not that big (so we will never run into that problem).

Reply:
That were our thoughts as well. One might adapt it so it shows sth. like 70:14 insted of just 10:14 (because the 1 hour is not shown)

Reply:
Yeah, that's probably a good idea (we just need to remove the modulo).